app/boards/shields/duckboard/duckboard.overlay

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ * SPDX-License-Identifier: MIT
+ *
+ * Author: Jack Hartstein
+ */
+
+#include "duckboard.dtsi"
```

Comment:
Since there's no left/right, etc nor need to re-use any common settings, please remove the `duckboard.dtsi` file and incorporate those DT items into this main overlay file.